go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -458,21 +459,26 @@ func (n *NodeController) ensureHybridOverlayBridge() error {
 
 	// Handle ARP for gateway address internally
 	portMACRaw := strings.Replace(portMAC.String(), ":", "", -1)
-	portIPRaw := getIPAsHexString(portIP)
-	_, stderr, err = util.RunOVSOfctl("add-flow", extBridgeName,
-		fmt.Sprintf("table=0,priority=100,in_port=%s,arp,arp_tpa=%s,"+
-			"actions=move:NXM_OF_ETH_SRC[]->NXM_OF_ETH_DST[],"+
-			"mod_dl_src:%s,"+
-			"load:0x2->NXM_OF_ARP_OP[],"+
-			"move:NXM_NX_ARP_SHA[]->NXM_NX_ARP_THA[],"+
-			"move:NXM_OF_ARP_SPA[]->NXM_OF_ARP_TPA[],"+
-			"load:0x%s->NXM_NX_ARP_SHA[],"+
-			"load:0x%s->NXM_OF_ARP_SPA[],"+
-			"IN_PORT",
-			rampExt, portIP.String(), portMAC.String(), portMACRaw, portIPRaw))
-	if err != nil {
-		return fmt.Errorf("failed to set up hybrid overlay bridge ARP flow,"+
-			"stderr: %q, error: %v", stderr, err)
+	for _, portIP := range portIPs {
```

Comment:
going to need to hold on this patch and rebase after the external gw stuff goes in